app/react/socket.js

```diff
@@ -1,17 +1,17 @@
 import io from 'socket.io-client';
 import { isClient } from 'app/utils';
 
-let socket = { on: () => {} };
+let _socket = { on: (_event, _listener) => {}, off: (_event, _listener) => {} };
```

Comment:
Why the underscored variable name?

Reply:
Because the parameter is not being used in the body, so eslint will complain, but we need them for the arity of the function to be correctly infered on ts files.